modules/monitoring-configuring-persistent-storage.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * monitoring/configuring-the-monitoring-stack.adoc
+
+:_content-type: CONCEPT
+[id="configuring_persistent_storage_{context}"]
+= Configuring persistent storage
+
+Running cluster monitoring with persistent storage means that your metrics are stored to a persistent volume (PV) and can survive a pod being restarted or recreated. This is ideal if you require your metrics or alerting data to be guarded from data loss. For production environments, it is highly recommended to configure persistent storage. Because of the high IO demands, it is advantageous to use local storage.
```

Comment:
Please review the "Claims and recommendations" section of the IBM Style Guide, in particular the subsection on "Product performance" which states 
```
Instead of writing that a certain utility "improves" performance, state that it
"can improve" performance or that it "is designed to improve" performance. Remember that performance testing does not reflect every customer environment.
```

Reply:
```suggestion
Running cluster monitoring with persistent storage means that your metrics are stored to a persistent volume (PV) and can survive a pod being restarted or recreated. This is ideal if you require your metrics or alerting data to be guarded from data loss. In production environments that have high IO demands, consider using persistent storage based on local volumes to improve performance.
```